src/Aspire.Hosting/Dcp/DcpExecutor.cs

```diff
@@ -174,6 +180,11 @@ public async Task StopAsync(CancellationToken cancellationToken)
         }
     }
 
+    public async ValueTask DisposeAsync()
+    {
+        await this.StopAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
nit: Unless it is nessessary.
```suggestion
        await StopAsync(CancellationToken.None).ConfigureAwait(false);
```